tests/tests.rs

```diff
@@ -1420,12 +1420,12 @@ fn test_exec_batch() {
 
         te.assert_failure_with_error(
             &["foo", "--exec-batch", "echo", "{/}", ";", "-x", "echo"],
-            "error: The argument '--exec <cmd>' cannot be used with '--exec-batch <cmd>'",
+            "error: The argument '--exec-batch <cmd>...' cannot be used with '--exec <cmd>...'",
```

Comment:
Hm. This is unexpected. `--exec-batch` can only occur once, IIRC. Maybe it now needs to be marked as `.multiple_occurences(false)`? Not sure why this wasn't required before.

Reply:
It still can't occur more than once:

![image](https://user-images.githubusercontent.com/2541726/148519623-63ec5720-7ad3-4b85-9e06-e01be5727bb4.png)

I think the `...` just indicates it takes more than one argument. Although since it is a single word, it may be a little confusing. I can't find a way to improve the way clap presents it though. (maybe we should make a feature request upstream?)

Reply:
ahh, I see. In that case, I think it's fine.